presto-tests/src/main/java/com/facebook/presto/tests/H2QueryRunner.java

```diff
@@ -157,7 +158,9 @@ public void close()
     public MaterializedResult execute(Session session, @Language("SQL") String sql, List<? extends Type> resultTypes)
     {
         MaterializedResult materializedRows = new MaterializedResult(
-                handle.createQuery(sql)
+                handle.setSqlParser(new RawSqlParser())
+                        .setTemplateEngine((template, context) -> template)
```

Comment:
You could comment (here or at `RawSqlParser` declaration), why these seemingly non-std configuration calls are needed

Reply:
Good idea. Added a comment.